outputs.tf

```diff
@@ -7,5 +7,5 @@ output "dashboard_url" {
 }
 
 output "dashboard_pasword" {
```

Comment:
`password` is misspelled in the output name here. Really excited about this, thanks for all of the hard work!